ledger/participant-integration-api/src/test/suite/scala/platform/indexer/RecoveringIndexerSpec.scala

```diff
@@ -295,38 +298,46 @@ object RecoveringIndexerSpec {
     case Failure(exception) => resource.release().flatMap(_ => Future.failed(exception))
   }
 
-  class TestIndexer(resultsSeq: SubscribeResult*) {
+  final class TestIndexer(resultsSeq: SubscribeResult*)(implicit loggingContext: LoggingContext) {
+    private[this] val logger = ContextualizedLogger.get(getClass)
     private[this] val actorSystem = ActorSystem("TestIndexer")
     private[this] val scheduler = actorSystem.scheduler
 
     private[this] val results = resultsSeq.iterator
     private[this] val actionsQueue = new ConcurrentLinkedQueue[IndexerEvent]()
 
+    private def addAction(event: IndexerEvent): Unit = {
+      logger.info(s"New test event: $event")
```

Comment:
do we want to keep this?

Reply:
Yeah, I added this so if the test fails, you get useful log output.

If the test passes, Bazel swallows the output (by default), so no harm done.